klassentrenner-ui/src/app/index/index.component.html

```diff
@@ -3,9 +3,9 @@ <H1>Schulklassen sinnvoll aufteilen gegen Corona!</H1>
     <div class="alert alert-warning" role="alert" *ngIf="news !== ''">
         {{news}}
     </div>
-    <p>In der aktuellen Corona Pandemie stellen Schulklassen ein Infektionsrisiko dar. Dieses kann verringert werden indem immer nur die halbe Schulklasse Präsenzunterricht hat. Allerdings sollte die Aufteilung der Klasse in zwei Hälften nicht willkürlich erfolgen da sonst Freizeit-Kontakte der Schüler die Infektion von der einen Hälfte der Klasse in die andere tragen können.</p>
+    <p>In der aktuellen Corona-Pandemie stellen auch Schulklassen ein Infektionsrisiko dar. Dieses kann verringert werden indem abwechselnd immer nur die halbe Schulklasse Präsenzunterricht hat. Eine zufällige Aufteilung der Klasse ignoriert jedoch Freizeitkontakte und Kontakte durch z.B. gemeinsame Wahlkurse. In einer Simulationsstudie konnte gezeigt werden, dass die Minimierung von Kontakten zwischen den Gruppen die Infektionen von einer zur anderen Gruppe um bis zu 79% reduzieren kann.</p>
```

Comment:
could you add the link to the preprint here with 
<a href="https://www.medrxiv.org/content/10.1101/2020.11.30.20241166v1"> Simulationsstudie</a>